dev/Repeater/ItemTemplateWrapper.cpp

```diff
@@ -55,6 +55,13 @@ winrt::UIElement ItemTemplateWrapper::GetElement(winrt::ElementFactoryGetArgs co
         // no element was found in recycle pool, create a new element
         element = selectedTemplate.LoadContent().as<winrt::FrameworkElement>();
 
+        // Template returned null, so insert empty element to render nothing
+        if (!element) {
+            element = winrt::Rectangle();
+            element.as<winrt::Rectangle>().Width(0);
```

Comment:
nit: could avoid doing cast twice if you store it in a local value. 
auto rectangle = winrt::Rectangle();
rectangle.Width(0);
rectangle.Height(0);
element = rectangle;

Reply:
Fixed that with latest commit.